### PR TITLE
student get method

### PR DESCRIPTION
**Pull request by SanojanYogathas**
## Description
post implementation

### CODE REVIEW

File: src/main/java/com/example/demo/DemoApplication.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package com.example.demo;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+
```
Comment (lahiruchandika) [inline src/main/java/com/example/demo/DemoApplication.java]: remove this line

File: src/main/java/com/example/demo/student/Student.java [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.SequenceGenerator;
+import jakarta.persistence.Table;
+import jakarta.persistence.Transient;
+
+
+@Entity
+@Table
+
```
Comment (lahiruchandika) [inline src/main/java/com/example/demo/student/Student.java]: remove this

File: src/main/java/com/example/demo/student/Student.java [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+        sequenceName = "student_sequence",
+        allocationSize = 1       
+    )
+    
+    @GeneratedValue(
+        strategy = GenerationType.SEQUENCE,
+        generator = "student_sequence"     
+    )
+    
+
```
Comment (lahiruchandika) [inline src/main/java/com/example/demo/student/Student.java]: remove lines

File: .mvn/wrapper/maven-wrapper.properties [hunk 4]
```diff
@@ -0,0 +1,2 @@
+distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.8.6/apache-maven-3.8.6-bin.zip
```
Comment (lahiruchandika97) [inline .mvn/wrapper/maven-wrapper.properties:1]: remove this file

**Pull request merged**